apps/storybook/src/XAxisZoom.stories.tsx

```diff
@@ -23,36 +18,35 @@ const Template: Story<TemplateProps> = (args) => {
 };
 
 export const Default = Template.bind({});
-Default.args = {
-  disabled: false,
-  modifierKey: 'Alt', // Initialize to default value
+
+export const ModifierKey = Template.bind({});
+ModifierKey.args = {
+  modifierKey: ['Alt'],
+};
+
+export const MultipleModifierKeys = Template.bind({});
+MultipleModifierKeys.args = {
+  modifierKey: ['Control', 'Alt'],
 };
 
 export const Disabled = Template.bind({});
 Disabled.args = {
   disabled: true,
-  modifierKey: 'Alt', // Initialize to default value
-};
-
-export const ModifierKey = Template.bind({});
-ModifierKey.args = {
-  disabled: false,
-  modifierKey: 'Shift',
 };
 
 export default {
-  ...VisCanvasStoriesConfig,
-  title: 'Building Blocks/PanZoom/X Zoom',
-  parameters: {
-    ...VisCanvasStoriesConfig.parameters,
-    controls: {
-      include: ['disabled', 'modifierKey'],
-    },
+  title: 'Building Blocks/Interactions/XAxisZoom',
+  component: XAxisZoom,
+  parameters: { layout: 'fullscreen' },
+  decorators: [FillHeight],
+  args: {
+    modifierKey: [],
+    disabled: false,
```

Comment:
Setting default `args` even for optional props like `disabled` helps with ordering the controls in the Storybook UI.